modules/system/traits/AssetMaker.php

```diff
@@ -299,7 +299,7 @@ public function getAssetPath($fileName, $assetPath = null)
             return $fileName;
         }
 
-        return $assetPath . '/' . $fileName;
+        return Url::asset($assetPath . '/' . $fileName);
```

Comment:
@jaxwilko I think it also needs to be applied on line 299 right above this, since any asset starting with `/` will return early there instead of being passed through the Url::asset() method.

Reply:
Added in d635c581dd75bffa2f8ee2b6d3a1e3c14170a96b